qutebrowser/browser/browsertab.py

```diff
@@ -276,11 +279,24 @@ def __init__(self, tab, parent=None):
         # self.destroyed.connect(functools.partial(
         #     cfg.changed.disconnect, self.init_neighborlist))
 
+    def _on_url_changed(self):
+        # NOTE: If a new page is requested and the request is slow to
+        # return data then this can be called while the "old" page is
+        # still displayed.
+        if self._default_zoom_changed:
+            # TODO: If someone has changed the zoom level for this tab
+            # then navigates to a tab with a custom default zoom should
+            # we change the tab zoom or expect people to press '='?
```

Comment:
I think that we can expect people to press `=`, I personally like the behavior here.